boards/BMS/Src/App/App_CellMonitorsTemperatures.c

```diff
@@ -0,0 +1,55 @@
+#include "App_CellMonitorsTemperatures.h"
+#include "configs/App_AccumulatorConfigs.h"
+
+static float *internal_die_temperatures;
```

Comment:
I actually was going to convert the raw voltages for the die temperature to die temperatures in this class but since we're not doing that anymore I don't see why we can't just move these functions to Io_DieTemperatures.